workers/wfsfeature-harvester/src/main/java/org/fao/geonet/harvester/wfsfeatures/worker/EsWFSFeatureIndexer.java

```diff
@@ -75,6 +76,9 @@
     public static final String TREE_FIELD_SUFFIX = "_tree";
     public static final String FEATURE_FIELD_PREFIX = "ft_";
 
+    @Value("es.index.features")
+    private String index = "features";
+
     @Autowired
     private EsClient client;
```

Comment:
Multinode mode could you use the same Elasticsearch index for features (and possibly catalog and search stats). BTW we should maybe ask if we want to keep this mode and support it, but that another issue.